docs/en/operations/system-tables/tables.md

```diff
@@ -1,59 +1,65 @@
+-   `metadata_path` ([String](../../sql-reference/data-types/string.md)) - Path to the table metadata in the file system. 
 
--   `dependencies_table` (Array(String)) - Table dependencies ([MaterializedView](../../engines/table-engines/special/materializedview.md) tables based on the current table).
+-   `metadata_modification_time` ([DateTime](../../sql-reference/data-types/datetime.md)) - Time of latest modification of the table metadata.
 
--   `create_table_query` (String) - The query that was used to create the table.
+-   `dependencies_database` ([Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md))) - Database dependencies.
 
--   `engine_full` (String) - Parameters of the table engine.
+-   `dependencies_table` ([Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md))) - Table dependencies ([MaterializedView](../../engines/table-engines/special/materializedview.md) tables based on the current table).
 
--   `partition_key` (String) - The partition key expression specified in the table.
+-   `create_table_query` ([String](../../sql-reference/data-types/string.md)) - The query that was used to create the table.
 
--   `sorting_key` (String) - The sorting key expression specified in the table.
+-   `engine_full` ([String](../../sql-reference/data-types/string.md)) - Parameters of the table engine. 
 
--   `primary_key` (String) - The primary key expression specified in the table.
+-   `partition_key` ([String](../../sql-reference/data-types/string.md)) - The partition key expression specified in the table. 
 
--   `sampling_key` (String) - The sampling key expression specified in the table.
+-   `sorting_key` ([String](../../sql-reference/data-types/string.md)) - The sorting key expression specified in the table. 
 
--   `storage_policy` (String) - The storage policy:
+-   `primary_key` ([String](../../sql-reference/data-types/string.md)) - The primary key expression specified in the table. 
+
+-   `sampling_key` ([String](../../sql-reference/data-types/string.md)) - The sampling key expression specified in the table. 
+
+-   `storage_policy` ([String](../../sql-reference/data-types/string.md)) - The storage policy:
 
     -   [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engine-mergetree-multiple-volumes)
     -   [Distributed](../../engines/table-engines/special/distributed.md#distributed)
 
--   `total_rows` (Nullable(UInt64)) - Total number of rows, if it is possible to quickly determine exact number of rows in the table, otherwise `Null` (including underying `Buffer` table).
+-   `total_rows` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) - Total number of rows, if it is possible to quickly determine exact number of rows in the table, otherwise `Null` (including underying `Buffer` table). 
```

Comment:
```suggestion
-   `total_rows` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) - Total number of rows, if it is possible to quickly determine exact number of rows in the table, otherwise `NULL` (including underying `Buffer` table). 
```